tests/test_table.py

```diff
@@ -1107,26 +1107,26 @@ def test_cast_array_to_features_to_null_type():
 def test_embed_array_storage(image_file):
     array = pa.array([{"bytes": None, "path": image_file}], type=Image.pa_type)
     embedded_images_array = embed_array_storage(array, Image())
-    assert embedded_images_array.to_pylist()[0]["path"] is None
+    assert isinstance(embedded_images_array.to_pylist()[0]["path"], str)
```

Comment:
We can also check the filename correctness explicitly:

```suggestion
    assert isinstance(embedded_images_array.to_pylist()[0]["path"], str)
    assert embedded_images_array.to_pylist()[0]["path"] == "test_image_rgb.jpg"
```
(here and below)